docs/source/filters.rst

```diff
@@ -69,6 +69,8 @@ At the moment, the following filters are built-in:
 - **sha1sum**: Calculate the SHA-1 checksum of the content
 - **shellpipe**: Filter using a shell command
 - **sort**: Sort input items
+- **remove-duplicate-lines**: Remove duplicate lines (case sensitive). Optional `separator` filter
+  to control how input is split into lines, defaults to `\n'.
```

Comment:
Remove the `Optional `separator` filter to control how input is split into lines, defaults to `\n'.` part, as this should be in the details.